seqr/utils/elasticsearch/es_search.py

```diff
@@ -691,7 +695,8 @@ def _is_matched_sample(family_guid, sample):
 
     def _parse_compound_het_response(self, response):
         if len(response.aggregations.genes.buckets) > MAX_COMPOUND_HET_GENES:
-            raise Exception('This search returned too many compound heterozygous variants. Please add stricter filters')
+            from seqr.utils.elasticsearch.utils import InvalidSearchException
```

Comment:
this file can't be imported at the top of the file due to circular import issues

Reply:
Nope, it just makes a circular dependency elsewhere :(